packages/composer/src/components/Input/index.tsx

```diff
@@ -178,6 +190,7 @@ export default function Input({
         />
 
         <Hotkey
+          preventDefault
```

Comment:
If we're in a modal or something else that relies on `esc`, then this hotkey would close *everything*. Preventing default so that it only affects the open menu.